gatsby/nice-icons.d.ts

```diff
@@ -12,3 +12,16 @@ declare module "@nice-digital/icons/lib/Print" {
 
 	export default PrintIcon;
 }
+
+declare module "@nice-digital/icons/lib/ChevronUp" {
```

Comment:
If you merge master in you'll get both chevron down and up:

https://github.com/nice-digital/cks-gatsby/blob/aa0f55b4f0a3de5105a5310ab2ea5f3b180fa827/gatsby/nice-icons.d.ts#L17-L41